source/autocomplete/autocomplete.h

```diff
@@ -304,13 +304,12 @@ struct Autocomplete
         }
     };
 
-    std::vector<fl_quality> sort_and_truncate_by_score(std::vector<fl_quality> input, size_t nbmax) const {
-        sort_and_truncate(input, nbmax, [](fl_quality a, fl_quality b){return a.score > b.score;});
-        return input;
+    void sort_and_truncate_by_score(std::vector<fl_quality>& input, size_t nbmax) const {
+        sort_and_truncate(input, nbmax, [](const fl_quality& a, const fl_quality& b){return a.score > b.score;});
     }
 
-    std::vector<fl_quality> sort_and_truncate_by_quality(std::vector<fl_quality> input, size_t nbmax) const {
-        sort_and_truncate(input, nbmax, [](fl_quality a, fl_quality b){return a.quality > b.quality;});
+    std::vector<fl_quality> sort_and_truncate_by_quality(std::vector<fl_quality>& input, size_t nbmax) const {
+        sort_and_truncate(input, nbmax, [](const fl_quality& a, const fl_quality& b){return a.quality > b.quality;});
         return input;
```

Comment:
as you can see, it's not my code... I changed the one used by my part but not this one